### DISCUSSION TITLE
[gatsby-theme-minimal-blog]: MacOs Monterey accent color change

### DISCUSSION THREAD

**Discussion by ghost** [LekoArts/gatsby-themes#643] [Questions] [answered]
Hi, 

In the new MacOS, the accent color of a webpage is defined by the overall color of the page to create continuity, but for the minimal blog theme this becomes a clear contrast instead (see picture). Is there anything in the CSS to change so the color becomes the same as the page (either navy or white theme) instead (see example picture from Apple)

<img width="1667" alt="Capture d’écran 2021-08-24 à 14 03 31" src="https://user-images.githubusercontent.com/85317794/130613483-9518cfd6-08d2-4a57-9b86-5474335651ec.png">

<img width="1248" alt="Capture d’écran 2021-08-24 à 14 06 11" src="https://user-images.githubusercontent.com/85317794/130613498-8b244989-57ac-4221-8526-5b03102de2bd.png">

**Comment by ghost** [comment]
I think if there is a way to change the "--theme-ui-colors-primary: #9f7aea;" found in the html (not my expertise area) this could be it, but no such code on the files so how does themeui work, this is the mystery

**Comment by LekoArts** [comment, marked as answer]
This is a wild guess but is maybe it's taking the `theme_color` from the generated manifest file into account?
https://github.com/LekoArts/gatsby-themes/blob/master/examples/minimal-blog/gatsby-config.js#L72

It's the purple you're seeing. Just for the sake of testing it, change that color and see if anything changes 😅

**Comment by ghost** [comment]
Restarted the computer, fresh Github clone and IT WORKS.
For anyone else, Leko saved the day again, thanks a lot !

**Reply by LekoArts** [reply]
yey, great to hear :)

**Comment by LekoArts** [comment]
I've found the answer also here now: https://developer.apple.com/documentation/safari-release-notes/safari-15-beta-release-notes

> Added support for the theme-color meta tag to change the tab bar background and over-scroll area in macOS and iPadOS, and the status bar in iOS.
